### DISCUSSION TITLE
How to alias device configs?

### DISCUSSION THREAD

**Discussion by gregsheremeta** [zwave-js/zwave-js#2779] [Q&A]
It's common for two devices to be identical other than ID and branding. For example,
SKU 14296 = https://byjasco.com/ge-z-wave-plus-wall-smart-dimmer-light-almond-toggle-500s
is identical to
SKU 14295 = https://byjasco.com/ge-enbrighten-z-wave-plus-smart-dimmer-with-quickfittm-and-simplewiretm-toggle-white
other than product SKU, color, and slightly different branding ("GE Enbrighten" vs "Enbrighten")

14296 did not have an entry in zwavealliance or in zwavejs config, so I submitted #2778. But I basically just copied and pasted the json from packages/config/config/devices/0x0063/14295_46204_zw3011.json

Is there a better way? I haven't dig into the templating system yet, but perhaps I should be using that?

**Comment by gregsheremeta** [comment]
seems like templating is the way to do this. I see no references to `$import` in the GE/Jasco directory `0063`, and it looks like it's just copy/paste or mass import in there.

**Reply by AlCalzone** [reply]
I'll let @blhoward2 answer this, but AFAIK the GE/Jasco/Whatever devices very often just differ in the branding, so we've often put them into one file.

**Comment by blhoward2** [comment]
It the device finger print is identical (manufacturer, productType and productID) we can't distinguish them so they go in one file and both model numbers are added to the file. If they are distinguishable we generally put them in different files unless there is a good reason not to, like different model numbers for different frequencies (X-US and X-EU). 

I'm not aware of any based solely on color but I'd be fine combining those. To do that you'd just add the fingerprint to the file and expand the label "14295 / 14296".

**Reply by gregsheremeta** [reply]
ok, so "go in one file" means just add both model numbers to the label... I was going to ask how that was done. Thanks.

**Comment by disforw** [comment]
The SKU change was mainly a marketing ploy for most of their active products. The switches are a little slimmer, the leads are pre-stripped for “simple-wire” and the heat-sync flanges were removed. The core functionality (and programming) was not touched. There should not be separate files added, the productIDs should be added as additional to the existing.

**Reply by blhoward2** [reply]
This describes 70% of the Ge devices though. There is a limit to how many labels you can display so at some point we just split them apart even if the parameters are the same.
